helpers/test/colour-contrast.html

```diff
@@ -0,0 +1,71 @@
+<!doctype html>
+<html>
+
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+	<title>Focus Helpers test</title>
+	<script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+	<script src="/node_modules/mocha/mocha.js"></script>
+	<script src="/node_modules/chai/chai.js"></script>
+	<script src="/node_modules/@polymer/test-fixture/test-fixture.js"></script>
+	<script src="/node_modules/wct-mocha/wct-mocha.js"></script>
+	<script type="module" src="./colour-contrast-components.js"></script>
+</head>
+
+<body>
+
+	<test-fixture id="colour-fixture">
+		<template>
+			<d2l-test-colour-contrast>
+			</d2l-test-colour-contrast>
+		</template>
+	</test-fixture>
+
+	<script type="module">
+		import { isAccessible } from '../colour-contrast/accessibility.js';
+
+		describe('colour-contrast', () => {
+
+			let colourFixture;
+
+			beforeEach(async() => {
+				colourFixture = fixture('colour-fixture');
+				await colourFixture.updateComplete;
+			});
+
+			describe('accessibility', () => {
+
+				it('Correctly passes/fails colours for AAA rating', () => {
+					[
+						{ rating: 'AAA', result: true },
+						{ rating: 'AA', result: false },
+						{ rating: 'F', result: false }
+					].forEach(({ rating, result }) => {
+						const backgroundColour = colourFixture.getBackgroundColour(rating);
+						const foregroundColour = colourFixture.getForegroundColour(rating);
+
+						expect(isAccessible(backgroundColour, foregroundColour)).to.equal(result);
```

Comment:
For AAA, shouldn't you need to pass in an override for the minimum contrast? AAA requires 7:1 for normal text and 4.5:1 for large text vs. AA is 4.5:1 for normal and 3:1 for large.

Reply:
Whoops, was not aware of this. I'll fix :+1: